tasks/trace_agent.py

```diff
@@ -11,7 +11,12 @@
 from .go import deps
 
 BIN_PATH = os.path.join(".", "bin", "trace-agent")
-DEFAULT_BUILD_TAGS = ["netcgo", "secrets"]
+DEFAULT_BUILD_TAGS = [
+    "netcgo",
```

Comment:
We might need more here, just like https://github.com/DataDog/datadog-agent/blob/29a972e3c1fdce1246105c630f74302f9678fb53/tasks/agent.py#L25-L45. Let's consolidate on that before merging.

Reply:
we're only missing the `docker` build tag I think, required by the `tagger`

Reply:
Added.